tests/extension/helper_launch_lock_taken.phpt

```diff
@@ -1,32 +1,36 @@
 --TEST--
 Can't launch helper because the lock is taken
 --INI--
-datadog.appsec.helper_socket_path=/does/not/exist
+datadog.appsec.helper_runtime_path=/tmp
```

Comment:
It's better to have a dedicated directory for the tests to avoid conflict with a server running locally.